src/go/rpk/pkg/api/admin/admin.go

```diff
@@ -131,6 +131,15 @@ var rng = func() func(int) int {
 	}
 }()
 
+// GetLeaderID returns the broker ID of the leader of the Admin API
+func (a *AdminAPI) GetLeaderID() (*int, error) {
+	pa, err := a.GetPartition("redpanda", "controller", 0)
+	if err != nil {
+		return nil, err
+	}
+	return &pa.LeaderID, nil
```

Comment:
Perhaps if LeaderID is -1, this function should return nil for the leader ID?  That way caller doesn't have to understand convention of -1 meaning nil.

Or, assume that the caller probably can't proceed without a known leader, and synthesize an error in this case.

Reply:
If we want `nil` instead of `-1` for a leaderless partition, maybe the right place to do it is in partition, so we also get `nil` in `rpk topic describe`?

Reply:
ahh you mean like the json should return `leader: null` or something like that? that would be good